extensions/amp-consent/0.1/consent-config.js

```diff
@@ -64,6 +64,10 @@ export class ConsentConfig {
       config['clientConfig'] = this.getConfig_()['clientConfig'];
     }
 
+    if (this.getConfig_()['uiConfig']) {
```

Comment:
Could you add the uiConfig to the comment [here](https://github.com/ampproject/amphtml/blob/master/extensions/amp-consent/0.1/consent-config.js#L150)